src/client/App.js

```diff
@@ -1,19 +1,21 @@
 import React from "react";
-import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
-import TestComponent from "./components/TestComponent/TestComponent";
+import { BrowserRouter as Router, Link, Route, Switch } from "react-router-dom";
+import About from "../pages/About";
+import Home from "../pages/Home";
+import Meals from "../pages/Meals";
+import NavBar from "./components/NavBar";
+import MealsDetail from "./components/MealsDetail";
 
 function App() {
   return (
     <Router>
-      <Route exact path="/">
-        <p>This is the main page</p>
-      </Route>
-      <Route exact path="/about">
-        <p>This is the about page</p>
-      </Route>
-      <Route exact path="/test-component">
-        <TestComponent></TestComponent>
-      </Route>
+      <NavBar />

```

Comment:
i mean to wrap the components by MealsProvider here so all children could use the values